src/isArray.js

```diff
@@ -0,0 +1,10 @@
+const isArray = (input) => {
+  if (Array.isArray(input)) {
```

Comment:
This pattern stands out to me as a code smell (or opportunity to improve the code) - we're evaluating a boolean expression and returning `true` if it is `true`, and `false` otherwise.  We can make this more concise by simply returning the result of the boolean expression:
```es6
const isArray = input => Array.isArray( input )
```